docs/source/modules/socrates.rst

```diff
@@ -0,0 +1,209 @@
+|``stellar_constant``        | 1368.22  | :math:`Wm^{-2}`, solar constant, consistent with RRTM default                           |
++----------------------------+----------+-----------------------------------------------------------------------------------------+
+|``tidally_locked``          | False    | Tidally locked or not                                                                   |
++----------------------------+----------+-----------------------------------------------------------------------------------------+
+|``frierson_solar_rad``      | False    | Options for annual-mean incoming solar, as prescribed in Frierson's grey scheme         |
++----------------------------+----------+-----------------------------------------------------------------------------------------+
+|``del_sol``                 | 1.4      | Latitudinal variation of shortwave radiation, as prescribed in Frierson's grey scheme   |
++----------------------------+----------+-----------------------------------------------------------------------------------------+
+|``del_sw``                  | 0        |                                                                                         |
++----------------------------+----------+-----------------------------------------------------------------------------------------+
+|``input_planet_emissivity`` | 1.0      | Emissivity of surface. Defined as constant all over surface.                            |
++----------------------------+----------+-----------------------------------------------------------------------------------------+
+
+The following namelist variables set radiation time stepping and spatial sampling:
+
++----------------------------+---------------+--------------------------------------------------------------------------+
+| Name                       | Default       | Description                                                              |
++============================+===============+==========================================================================+
+| ``dt_rad``                 | 0             | Radiation timestep - every step if ``dt_rad<dt_atmos``                   |
++----------------------------+---------------+--------------------------------------------------------------------------+
+| ``store_intermediate_rad`` | True          | Keep rad constant over entire dt_rad?                                    |
++----------------------------+---------------+--------------------------------------------------------------------------+
+| ``dt_rad_avg``             | -1            | If averaging, over what time? ``dt_rad_avg=dt_rad`` if ``dt_rad_avg<=0`` |
++----------------------------+---------------+--------------------------------------------------------------------------+
+| ``chunk_size``             | 16            | Number of gridpoints to pass to socrates at a time                       |
```

Comment:
Perhaps you could add (See `README.md <https://github.com/ExeClim/Isca/blob/master/exp/test_cases/socrates_test/README.md>`_) to this description.